src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball randomBall = new Ball(Color.BLUE, 0);
```

Comment:
The `randomBall` is created but never used because it is immediately replaced in the loop starting at line 8. Consider removing it to clean up the code.

Reply:
I was sure that I need to create a ball to overwrite it in next step. Thank you for tip.